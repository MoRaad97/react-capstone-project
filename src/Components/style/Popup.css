.popup {
  position: fixed;
  top: 0;
  width: 100%;
  height: 100vh;
  background-color: hsl(0, 0%, 98%);
  z-index: 9;
}

.popup-df {
  background-color: #f076a5;
}

.popup > .container {
  padding: 0 15px;
  margin: auto;
}

.popup .close-btn {
  margin: 30px 0;
  font-size: 20px;
  background: white;
  border: none;
  padding: 6px 12px;
  box-shadow: 0 0 8px 1px hsl(0deg 0% 85%);
}

.popup .close-btn-df {
  background-color: #ec4c8a;
  box-shadow: 0 0 8px 1px #5a3d5c;
  color: white;
}

.popup .popup-card {
  background: white;
  border-radius: 5px;
  padding-bottom: 10px;
  box-shadow: 0 0 8px 1px hsl(0deg 0% 85%);
  max-width: 450px;
  margin: auto;
}

.popup .popup-card-df {
  background-color: #ec4c8a;
  box-shadow: 0 0 8px 1px #5a3d5c;
  color: white;
}

.popup .popup-card img {
  width: 100%;
  border-radius: 5px 5px 0 0;
  height: 175px;
}

@media (min-width: 768px) {
  .popup .popup-card {
    display: flex;
    justify-content: space-evenly;
    align-items: center;
    min-height: 80vh;
    margin: 0;
    max-width: none;
  }

  .popup .popup-card img {
    max-width: 45%;
    height: 300px;
  }

  .popup .popup-card .popup-data-container ul {
    display: flex;
  }
}

.popup .popup-card .popup-data-container {
  padding-left: 10px;
}

.popup .popup-card .popup-data-container h2 {
  margin: 20px 0;
  font-family: "Gill Sans", sans-serif;
  font-weight: 900;
}

.popup .popup-card .popup-data-container ul {
  margin: 15px 0 10px;
  padding: 0;
  list-style: none;
  font-family: "Lato", sans-serif;
}

.popup .popup-card .popup-data-container ul li {
  font-size: 19px;
  margin-bottom: 11px;
}
