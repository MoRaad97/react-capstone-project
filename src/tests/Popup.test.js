import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/Store';
import Popup from '../Components/popup';

it('popup Component snapshot test', () => {
  const PopupRender = TestRenderer
    .create(
      <Provider store={store}>
        <Popup />
      </Provider>,
    )
    .toJSON();
  expect(PopupRender).toMatchSnapshot();
});