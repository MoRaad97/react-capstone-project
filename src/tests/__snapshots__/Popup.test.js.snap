// Jest Snapshot v1, https:
//goo.gl/fbaqlp
exports[`popup component snapshot test 1`] = `null`;JestSnapshotv1,https
